hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/embedded/RepeatableEmbeddedHedera.java

```diff
@@ -93,6 +95,14 @@ public TransactionResponse submit(
         return response;
     }
 
+    /**
+     * Causes the next transaction submitted to a non-default node to be encoded as
+     * a {@link Transaction} message, as before HIP-1127.
+     */
+    public void bypassNextWithPreHip1127TxFormat() {
```

Comment:
Is it expected that this method is only called from test code?

Reply:
Yes this is in `test-clients` module @artemananiev